recipe/meta.yaml

```diff
@@ -1,62 +1,95 @@
 # version numbering is explained at https://stackoverflow.com/questions/42730478/version-numbers-for-visual-studio-2017-boost-and-cmake
 #     This version number follows the "platform toolset"
-{% set vcver="14" %}
-{% set vsyear="2015" %}
-{% set fullver="14.0.25420" %}
+{% set vcver = "14.1" %}
+{% set vsyear = "2017" %}
+{% set fullver = "14.16.27012" %}
+# VS2017 is fundamentally compatible with VS2015.  We name our package vs2015_runtime so that it can't be
+#    mixed up with the runtime from VS2015 - you do actually need VS2017 runtime for things compiled with
+#    VS2017, and the runtime we have here is backwards-compatible with things compiled with VS2015.
+{% set runtime_year = "2015" %}
 
 package:
   name: vs{{ vsyear }}
   version: {{ fullver }}
 
 build:
-  number: 11
```

Comment:
This could be reset to 0 since the package name a version changed.

Reply:
done